miso-web/src/main/webapp/scripts/list_study.js

```diff
@@ -31,14 +31,14 @@ ListTarget.study = {
     if (config.isAdmin) {
       return [{
         "name": "Delete",
-        "action": function(ids) {
+        "action": function(studies) {
           if (confirm("Are you sure you really want to delete? This operation is permanent!")) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this usage of confirm(...). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1442)